sloppy-deployment/entrypoint.sh

```diff
@@ -1,6 +1,16 @@
 #!/bin/bash -eu
 
+# Name of the Sloppy YAML configuration file
 SLOPPY_OUTPUT_FILE="sloppy.yml"
+# Name of the directory (which should be host volume) to where a copy of
+# $SLOPPY_OUTPUT_FILE is saved.
+SLOPPY_SAFE_OUTPUT_DIR="${SLOPPY_SAFE_OUTPUT_DIR:-}"
```

Comment:
Please change `SLOPPY_SAFE_OUTPUT_DIR` finally to `SLOPPY_SAVE_OUTPUT` as requested 😉 The typo is using `SAFE` instead of `SAVE`: Please see https://hinative.com/de/questions/3352646 for further explanation.

Reply:
s/safe/save/ groked (at last) and fixed.